group_vars/all.yml.sample

```diff
@@ -45,6 +45,7 @@ dummy:
 #cluster: ceph
 
 # Inventory host group variables
+#all_group_name: all
```

Comment:
I don't think we need to define this because the `'all'` group name is an ansible builtin.